apps/client/src/app/qr/page.tsx

```diff
@@ -0,0 +1,18 @@
+'use-client'
+
+import QRCode from "react-qr-code";
+
+export default function Id() {
+    const UserId = '12345';
+    return <div className='m-10'>
```

Comment:
Can we use padding in this case? Margin is usually used between some elements. Usually when we want to add spacing around element, we use padding.

Reply:
Definitly :)  you are right